app/src/main/kotlin/com/saltedge/authenticator/features/authorizations/list/AuthorizationsListInteractorV2.kt

```diff
@@ -168,10 +169,11 @@ class AuthorizationsListInteractorV2(
 
     private fun processEncryptedAuthorizationsResult(encryptedList: List<AuthorizationResponseData>) {
         contract?.coroutineScope?.launch(defaultDispatcher) {
-            val splitList = encryptedList.partition { it.status.isFinalStatus }
-            val finishedData = prepareFinishedAuthorizationData(splitList.first)
+            val notClosedItems = encryptedList.filterNot { KEY_STATUS_CLOSED == it.status  }
```

Comment:
maybe add extension `isClosed` to `AuthorizationStatus`? And it will look like: `{ it.status.isClosed }`